### PR TITLE
Adding new operation

### PR DESCRIPTION
**Pull request by cs-tushar-kanade**
Adding new  connector operation to convert simple JSON file to CSV file.
This CSV file is created as attachment in FortiSOAR
https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=21237

### CODE REVIEW

**Comment by cs-tushar-kanade** [comment]
One query, right now its simple JSON which we read. I can create sub routine to read nested JSON. Should I take input from user to state if JSON is nested or simple ?

**Comment by cs-pooja-singh** [comment]
Can we detect that on our own @cs-tushar-kanade  ? In most cases, user might not know, or could get both types

File: csv-data-management/extract_data_from_csv.py [hunk 1]
```diff
@@ -425,6 +451,7 @@ def _df_to_csv(df):
     df.to_csv('/tmp/{}'.format(file_name), encoding='utf-8', header='true',compression="zip")
     filepath = '/tmp/{}'.format(file_name)
     ch_res = create_cyops_attachment(filename=filepath,name=file_name,description='Create by CSV Data Management Connector')
+    remove(filepath)
```
Comment (cs-pooja-singh) [inline csv-data-management/extract_data_from_csv.py]: can we put the remove in an except block so that file gets removed even if create_attachment fails

Comment (cs-tushar-kanade) [reply csv-data-management/extract_data_from_csv.py]: Done

**Comment by cs-tushar-kanade** [comment]
> 



> 

We can check nesting but if JSON is reset I need to ask user for key which will act as root and then ask for keys which will be  columns for CSV.

**Review by cs-pooja-singh — APPROVED** [review]

**Pull request merged**